SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -510,6 +510,11 @@ void USpatialNetDriver::NotifyActorDestroyed(AActor* ThisActor, bool IsSeamlessT
 
 void USpatialNetDriver::OnOwnerUpdated(AActor* Actor)
 {
+	if (!IsUsingQBI())
```

Comment:
Yeah so I'm thinking, this doesn't need to be here since it invokes a small amount of work (which I think is totally fine this is temporary). So in that case `IsUsingQBI` doesn't have to be a function and the `if` in the `CreateInterest` function just gets the bool directly.